network-api/networkapi/wagtailpages/pagemodels/libraries/rcc/landing_page.py

```diff
@@ -32,16 +32,14 @@ class RCCLandingPage(BasePage):
     content_panels = wagtail_models.Page.content_panels + [
         FieldPanel("intro"),
         FieldPanel("banner_image"),
-        # TODO: Reactivate once links are implemented
-        # InlinePanel("featured_content_types", heading="Featured content types"),
+        InlinePanel("featured_content_types", heading="Featured content types"),
     ]
 
     translatable_fields = [
         TranslatableField("title"),
         SynchronizedField("banner_image"),
         TranslatableField("intro"),
-        # TODO: Reactivate once links are implemented
-        # TranslatableField("featured_content_types"),
+        TranslatableField("featured_content_types"),
```

Comment:
Again, I'm just noticing this in this PR but I think the import for this should change and we should be using something like

```python
from wagtail_localize import fields as localize_fields

localize_fields.TranslatableField
```